src/Saas.Admin/readme.md

```diff
@@ -8,70 +8,74 @@ The SaaS Admin Service is an API that is reponsible for tenant management operat
+- A connection string to a running, empty SQL Server Database.
+    - [Local DB](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/sql-server-express-localdb?view=sql-server-ver15) (Windows Only) - See `Additional Resources` below for basic config secret
     - [SQL Server Docker Container](https://hub.docker.com/_/microsoft-mssql-server)
     - [SQL Server Developer Edition](https://www.microsoft.com/en-us/sql-server/sql-server-downloads)
+- A user store compatible with Microsoft Identity
+    - [Azure AD B2C](https://azure.microsoft.com/en-us/services/active-directory/external-identities/b2c/) - created automatically with Bicep deployment
 
-### App Settings
-
-In order to run the project locally, the App Settings marked as `secret: true` must be set using the [.NET secrets manager](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows). When deployed to azure using the bicep deployments, these secrets are [loaded from Azure Key Vault](https://docs.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-6.0#secret-storage-in-the-development-environment) instead.
-
-Default values for non secret app settings can be found in [appsettings.json](Saas.Admin.Service/appsettings.json)
+### ii. Development Tools
 
-| AppSetting Key |  Description | Secret | Default Value |
-| ---  | --- | --- | --- |
-| AzureAdB2C:Instance | | true |  |
-| AzureAdB2C:Domain | | true |  |
-| AzureAdB2C:ClientId | | true |  |
-| AzureAdB2C:TenantId | | true |  |
-| AzureAdB2C:SignedOutCallbackPath | | false | /signout/B2C_1A_SIGNUP_SIGNIN |
-| AzureAdB2C:SignUpSignInPolicyId | | false | B2C_1A_SIGNUP_SIGNIN |
-| KeyVault:Url | KeyVault URL to pull secret values from in production | false |  |
-| KeyVault:PermissionsApiCert | The name of the secret in Azure Key Vault that contains a base64 encoded certificate to use for authentication with the permissions api | false |  |
-| PermissionsApi:BaseUrl | URL for downstream [Permissions API](../Saas.Identity/Saas.Permissions/readme.md) | false | |
-| PermissionsApi:LocalCertificate | A Base64 encoded certificate (.CER) used to authenticate with the permissions API. Only used for local development. | true | |
-| ConnectionStrings:TenantsContext | Connection String to SQL server database used to store tenants data. If using local db for development, this connection string is fine to commit to your repo as it does not contain any secrets. | true | (local db connection string) |
-| AllowedHosts | | false |  * |
-| Logging:LogLevel:Default | | false | Information  |
-| Logging:LogLevel:Microsoft.AspNetCore | | false | Warning  |
+- [NSwag](https://github.com/RicoSuter/NSwag) - An NSwag configuration file has been included to generate an appropriate client from the included Admin project.
+    *Consumes Clients:*
+	- [permissions-service-client-generator.nswag](Saas.Admin.Service/permissions-service-client-generator.nswag)
```

Comment:
I think these files might have gotten misplaced. 

The permissions service client should probably be relocated to live with the permissions service api. 

I think the admin service client was put in the signupadmin site (https://github.com/Azure/azure-saas/blob/main/src/Saas.SignupAdministration/Saas.SignupAdministration.Web/admin-service-client-generator.nswag). This should probably be located in the saas.admin api folder. This is the one you're looking for here, not the permissions service one.

Reply:
External conversation confirmed this setup was correct, no changes necessary.